spring-boot-project/spring-boot-actuator-autoconfigure/pom.xml

```diff
@@ -188,6 +188,16 @@
 			<artifactId>commons-dbcp2</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
```

Comment:
We try to keep dependencies in alphabetical order (by groupId and then by artifactId)